src/App.svelte

```diff
@@ -1,82 +1,74 @@
 <script lang="ts">
-	import Counter from "./counter.svelte";
-	import { counterItems } from "./store"
+  import Counter from "./counter.svelte";
 
-	interface CounterItems {
-		countername: string;
-		count: number;
-	}
+  interface CounterItems {
+    countername: string;
+    count: number;
+  }
 
-	function addCounter() {
-    	$counterItems = [...$counterItems, {countername: "new", count: 0}]
-  	}
-	
-	function sumCount(Items: CounterItems[]) {
-		let sumcount: number = 0;
-		Items.forEach(element => {
-			sumcount += element.count;
-		});
-		return sumcount;
-	}
+  let counterItems: CounterItems[] = [{ countername: "new", count: 0 }];
 
-	$: sum = sumCount($counterItems);
+  function addCounter() {
+    counterItems = [...counterItems, { countername: "new", count: 0 }];
+  }
+
+  $: titlelist = counterItems.map((Item) => {
+    return Item.countername;
+  });
+
+  $: sum = counterItems
+    .map((Item) => {
+      return Item.count;
+    })
+    .reduce((previousValue, currentValue) => previousValue + currentValue, 0);
```

Comment:
こちらは、mapを使わなくても、reduceだけでいいと思いました 🙆‍♂️ 

```suggestion
  $: sum =  countersItems.reduce((sum, current) => sum + current.count, 0)
```
```